docs/api/wrapper/get.md

```diff
@@ -0,0 +1,19 @@
+## get
+
+Works just like [find](../find.md) but will throw an error if selector not
```

Comment:
Can be worded a little better - "but will throw an error if nothing matching the given selector is found".

Reply:
I made that change.